mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -272,6 +272,16 @@
     <!-- Content description (not visible, for screen readers etc.): Close button for the home onboarding dialog -->
     <string name="onboarding_home_content_description_close_button">Close</string>
 
+    <!-- Notification pre-permission dialog -->
+    <!-- Enable notification pre permission dialog title -->
+    <string name="onboarding_home_enable_notifications_title" tools:ignore="UnusedResources">Notifications help you do more with Firefox</string>
```

Comment:
"Firefox" should be a placeholder, and comment should explain what placeholder will be replaced with at run time.

Reply:
Sorry about this.  I'll fix.  Thanks

Reply:
Update landed.  Thanks!